tests/runtests.py

```diff
@@ -171,6 +177,22 @@ def setup_additional_options(self):
             action='store_true',
             help='Do not run any tests. Simply start the daemons.'
         )
+        self.add_option(
+            '--custom',
+            default=False,
+            action='store_true',
+            help='Addtional directories to scan for tests'
```

Comment:
This help text doesn't appear to reflect help this option? Maybe `Run only tests found in the directory passed by --custom-dir`?